### PR TITLE
add one more comment

### PR DESCRIPTION
**Pull request by mdr-datascience**

### CODE REVIEW

File: runfit.m [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Use this file to fit a curve using MATLAB
+
+# Load data
```
Comment (mjduarterosa) [inline runfit.m:3]: Provide more information in comment (e.g. what's the format of the data)

Comment (mjduarterosa) [inline runfit.m:3]: Also maybe turn code into function

**Review by mjduarterosa — CHANGES_REQUESTED** [review]
In general good but please fix issues in comments.

File: runfit.m (hunk 1 above)
Comment (mjduarterosa) [inline runfit.m:3]: Add more information as docstring to function

**Review by mjduarterosa — CHANGES_REQUESTED** [review]
Added one more suggestion to be reviewed

**Review by mdr-datascience — COMMENTED** [review]
All issues addressed

**Pull request merged**